### PR TITLE
Events summer

### PR DESCRIPTION
**Pull request by stefaniebutland**
fixes https://github.com/Openscapes/how_we_work/issues/358

### CODE REVIEW

**Comment by stefaniebutland** [comment]
not sure why BMIC event commit not showing here

**Comment by stefaniebutland** [comment]
> not sure why BMIC event commit not showing here.
I needed to push my commits. lol

**Comment by stefaniebutland** [comment]
@jules32 Ready for your review. I did not push the `_site` files to this PR yet. Am I correct in remembering that this is a way to avoid some merge conflicts, and that you will pull the PR and render locally to preview it?

File: events/2024-08-07-nih-bmic-talk/index.qmd [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+description: "Invited virtual talk for the NIH Biomedical Informatics Coordinating Committee (BMIC)" 
+author: Openscapes
+date: 2024-08-07
+categories:
+  - talk
+  - event
+image: bmic.png
+---
+
+*This event will follow up on and share from the* [*NLM Workshop on advancing a networked approach to equitable open science*](https://openscapes.org/events/2024-06-04-nlm-equitable-open-science)*. Maryam Zaringhalam, workshop lead, will give a report on the NLM workshop, and Julia Lowndes will provide a perspective on the approach Openscapes has developed to shift research cultures towards greater colla*boration and inclusion through open science. *Maryam Zaringhalam recently gave a keynote summarizing the NLM workshop at a CZI Open Science Meeting ([abstract & slides](https://zenodo.org/records/12523362)).*
```
Comment (jules32) [inline events/2024-08-07-nih-bmic-talk/index.qmd:12]:
```suggestion
This event will follow up on and share from the [NLM Workshop on advancing a networked approach to equitable open science](https://openscapes.org/events/2024-06-04-nlm-equitable-open-science). Maryam Zaringhalam, workshop lead, will give a report on the NLM workshop, and Julia Lowndes will provide a perspective on the approach Openscapes has developed to shift research cultures towards greater collaboration and inclusion through open science. Maryam Zaringhalam recently gave a keynote summarizing the NLM workshop at a CZI Open Science Meeting ([abstract & slides](https://zenodo.org/records/12523362)).
```

**Review by jules32 — APPROVED** [review]
This all looks great to me! Just some small edits around italics

**Comment by jules32** [comment]
Hi @stefaniebutland ! I did build it locally, so fine that you didn't push the _site files. I think in the future it's also ok if you build them and then I can pull to avoid conflicts if I test/build locally. 

Ready for merging, thank you!

**Pull request merged**